### DISCUSSION TITLE
Hysteresis Examples seem broken on my setup.

### DISCUSSION THREAD

**Discussion by taylorh140** [OpenModelica/OpenModelica#9494] [Q&A] [answered]
I seem to be having a weird problem with open Modelica. (great program btw). 
When I try to run the Hysteresis examples on my work computer they fail with a divide by zero notice. **I assume this is not an issue with the code but something incorrectly set up with my individual machine.** 
(it works fine on my computer at home which I believe is also v 1.19.2. )

I have uninstalled and reinstalled the software but got the same result. Is there something I need to do to clean the environment? 
![Screenshot 2022-10-07 132942](https://user-images.githubusercontent.com/10996543/194625348-465ce47c-d48c-4e9a-8a4e-edadd19fda15.png)

Attempting to simulate this gives:

![Screenshot 2022-10-07 133101](https://user-images.githubusercontent.com/10996543/194625609-d28ee201-905e-4789-869d-5187d9a314fb.png)

I have removed %Temp%/.openmodelica folder between reinstalls but the problem persists.

**Comment by casella** [comment, marked as answer]
Any settings you made with Tools | Options are retained from one installation to the next. You can get back to factory settings by clicking on the Reset button in that window.

**Comment by taylorh140** [comment]
This fixed everything. Thank you very much. Maybe it should be an option with the installation, that would have been helpful.

**Reply by casella** [reply]
Do you mean an option to reset to factory settings during installation? That could be a good idea
